src/pylexibank/commands/check_cognates.py

```diff
@@ -0,0 +1,44 @@
+"""
+Check cognates table
+"""
+import functools
+from collections import defaultdict
+from cldfbench.cli_util import with_datasets, add_catalog_spec
+from pylexibank.cli_util import add_dataset_spec, warning
+
+
+def register(parser):
+    add_dataset_spec(parser, multiple=True)
+    add_catalog_spec(parser, 'glottolog')
+
+def check(ds, args):
+    warn = functools.partial(warning, args, dataset=ds)
+    cldf = ds.cldf_reader()
+    
+    if cldf["CognateTable"].common_props.get('dc:extent') == 0:
+        # NOTE: this is using the metadata to identify if we
+        # have cognates or not. Is there a better way?
+        return
```

Comment:
Since we are going to loop over the cognates table lateron anyway, we could also use this to determine if there are any cognates.